src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -80,16 +87,28 @@ void uct_ib_mlx5_parse_cqe_zipping(uct_ib_mlx5_md_t *md,
                                    const uct_ib_mlx5_iface_config_t *mlx5_config,
                                    uct_ib_iface_init_attr_t *init_attr)
 {
-    if (!mlx5_config->cqe_zipping_enable) {
-        return;
-    }
+    uct_ib_dir_t dirs[UCT_IB_DIR_NUM] = {};
```

Comment:
Why is it of type `uct_ib_dir_t`?
Seems that the type of each array member is "bool"

Reply:
WDYT of having a bitmap instead of an array here,
then using something like `UCS_BIT(dir_idx)` in the `for` loop below?